variables.tf

```diff
@@ -97,7 +97,7 @@ variable "external_database_host" {
 }
 
 variable "enable_vpc_endpoint_services" {
-  description = "Enable metastore VPC endpoint services,for cross-account access."
+  description = "Enable metastore NLB,Route53 entries VPC access and VPC endpoint services,for cross-account access."
```

Comment:
Can you make the same documentation change to VARIABLES.md?

Reply:
done